gateway-openapi-ui/src/main/resources/swagger/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "swagger-ui-dist",
+  "version": "3.36.2",
+  "main": "index.js",
+  "repository": "git@github.com:swagger-api/swagger-ui.git",
+  "contributors": [
+    "(in alphabetical order)",
+    "Anna Bodnia <anna.bodnia@gmail.com>",
+    "Buu Nguyen <buunguyen@gmail.com>",
+    "Josh Ponelat <jponelat@gmail.com>",
+    "Kyle Shockey <kyleshockey1@gmail.com>",
+    "Robert Barnwell <robert@robertismy.name>",
+    "Sahar Jafari <shr.jafari@gmail.com>"
+  ],
```

Comment:
Is this required? Does this need to be kept up to date?

Reply:
Removed.